openmetrics/src/main/java/io/airlift/openmetrics/MetricsResource.java

```diff
@@ -319,7 +333,17 @@ private Stream<Metric> getManagedMetricsStream()
         Map<String, ManagedClass> managedClasses = this.mbeanExporter.getManagedClasses();
```

Comment:
These are not guaranteed to be jmx mbeans

Reply:
TIL that are can be non-jmx mbeans :D Can we filter them somehow?